src/components/override-sdk/template/Details/Details.tsx

```diff
@@ -28,15 +29,7 @@ export default function Details(props) {
     // Conditionally wrap in main wrapper only if we are not in a case with and open status (i.e. we are in a finished case, viewing the claim summary)
     <ConditionalWrapper    
       condition={!PCore.getStore().getState().data[containerName].caseInfo?.status.startsWith('Open')}
-      wrapper = {childrenForWrap =>        
-      <main className="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
-        <div className="govuk-grid-row">
-          <div className='govuk-grid-column-two-thirds'>
-            {childrenForWrap}
-          </div>
```

Comment:
Please check here that main wrapper is not duplicated (especially in the Submitted Claim review page, where child details are displayed)